UnitedIncome/components
# [APP-3225] Font Sizing Helper Classes

### Description
> Please provide a brief description of your changes below.

Adjusts the appropriate helper font sizing styles to match the mobile comps for dotcom. The 1.5 is technically h2, which is why the numbers get thrown off slightly.

These font sizing considerations are made using this base;

```css
html {
  font-size: 10px;
}

@media (max-width: 991px) {
  html {
    font-size: 8px;
  }
}
```

<img width="1144" alt="Screen Shot 2019-08-19 at 10 06 19 AM" src="https://user-images.githubusercontent.com/10888441/63272273-c55be780-c269-11e9-8e6f-de2b38980f4c.png">

### Testing Instructions
> What should be done in order to test this pull request, or re-create its effects.

* Observe the mobile font sizing.
* It should match the comps attached under the `Comments` section.

### Comments
> Any additional comments that will aid with the review process.

- [JIRA Ticket](https://unitedincome.atlassian.net/browse/app-3225)
- [Zeplin Comps](https://app.zeplin.io/project/5c9d630d57fbe3581e5c889e/screen/5d559adf0ed683524706ba14)
